Fix the cluster terminology for v5 (#144)
neo4j/docs-getting-started: modules/ROOT/pages/get-started-with-neo4j/index.adoc

[[neo4j-first-steps]]
== First steps with Neo4j

> 1. If your purpose is to learn Neo4j graph database features, how to write Cypher queries, use developer tools like xref:get-started-with-neo4j/graph-platform.adoc#neo4j-tools[Neo4j Browser and Bloom] and get an overview on how to connect to Neo4j via drivers and connectors, go to the link:https://neo4j.com/cloud/platform/aura-graph-database/[AuraDB page], where you can create a Neo4j instance free forever. +

Reviewer: Also, unsure if this is true? Is the source of this link really the place we want to send people?

> 1. If your purpose is to learn Neo4j graph database features, how to write Cypher queries, use developer tools like xref:get-started-with-neo4j/graph-platform.adoc#neo4j-tools[Neo4j Browser and Bloom] and get an overview on how to connect to Neo4j via drivers and connectors, go to the link:https://neo4j.com/cloud/platform/aura-graph-database/[AuraDB page], where you can create a Neo4j instance free forever. +

Author: https://neo4j.com/cloud/platform/aura-graph-database/ is the Aura product  page. You see subscription plans there. Button 'Start Free' is one of the first things that grab your attention

> 1. If your purpose is to learn Neo4j graph database features, how to write Cypher queries, use developer tools like xref:get-started-with-neo4j/graph-platform.adoc#neo4j-tools[Neo4j Browser and Bloom] and get an overview on how to connect to Neo4j via drivers and connectors, go to the link:https://neo4j.com/cloud/platform/aura-graph-database/[AuraDB page], where you can create a Neo4j instance free forever. +

Reviewer: Right, my concern is whether this is the best place to start?

> 1. If your purpose is to learn Neo4j graph database features, how to write Cypher queries, use developer tools like xref:get-started-with-neo4j/graph-platform.adoc#neo4j-tools[Neo4j Browser and Bloom] and get an overview on how to connect to Neo4j via drivers and connectors, go to the link:https://neo4j.com/cloud/platform/aura-graph-database/[AuraDB page], where you can create a Neo4j instance free forever. +

Author: I think , yes. What entry-point to AuraDB could you recommend?

> 1. If your purpose is to learn Neo4j graph database features, how to write Cypher queries, use developer tools like xref:get-started-with-neo4j/graph-platform.adoc#neo4j-tools[Neo4j Browser and Bloom] and get an overview on how to connect to Neo4j via drivers and connectors, go to the link:https://neo4j.com/cloud/platform/aura-graph-database/[AuraDB page], where you can create a Neo4j instance free forever. +

Reviewer: Let's discuss this afternoon during our workshop.

> AuraDB is the best choice for cloud developers who are interested in graph technologies. +

Reviewer: ```suggestion
AuraDB is a good choice for cloud developers who are interested in graph technologies. +
```
